src/components/views/messages/EditHistoryMessage.js

```diff
@@ -115,16 +126,36 @@ export default class EditHistoryMessage extends React.PureComponent {
         );
     }
 
+    _diffIt(oldBody, newBody) {
```

Comment:
Nit: Maybe just `_diff`? Unsure why this method would use an `It` suffix...

Reply:
Whoops, forgot to rename after prototype code :)